snapshot/db.go

```diff
@@ -0,0 +1,45 @@
+package snapshot
+
+// ID identifies a value in DB
```

Comment:
if 'value' is a snapshot the comment should use the word 'snapshot' otherwise, we need more info about what a 'value' is.